Classes/AMPPreviewController.h

```diff
@@ -43,6 +43,12 @@ typedef void (^AMPPreviewControllerFinishDownload)(NSError *error);
 
 @interface AMPPreviewController : QLPreviewController
 
+/**
+ The URL of the destination directory for the downloaded assets from
+ remoteURL
+ */
+@property (nonatomic, strong, readonly, class) NSURL *tempRemoteURLFilesDirectory;
```

Comment:
what's the purpose of this?

Reply:
Exposing this to external clients so that they can clean it whenever convenient for them

Reply:
That makes sense. But to be honest this being iOS we know that what the temp folder is, and we're not making a subdirectory anyway, so we could just document that and let the clients clean NSTemporaryDirectory if they want to. Without making a subdirectory for this Pod I don't see how useful this is, since the client will just endup cleaning the entire temp directory anyway.

Reply:
Yes, but I think this should be kept as an implementation detail to give room in the future to improve that and add a proper cache that isn't necessarily the temp directory.